issuesync.rb

```diff
@@ -226,6 +229,14 @@ def updated_at() Time.parse(data['updated_at']) end
     def has_comments?() !data['comments'].zero? end
     def open?() data['state'] == 'open' end
     def closed?() !open? end
+
+    def labels()
```

Comment:
Thanks - this is my first ever foray into Ruby, so I was mostly copying. That said, I see `has_comments?`/`open?`/`closed?` on the line above all have parens. Are they required for some other reason? Or is this a style preference/convention that has changed?